test/module/irohad/ametsuchi/kv_storage_test.cpp

```diff
@@ -107,7 +108,8 @@ class KVTest : public AmetsuchiTest {
           [](iroha::expected::Error<std::string> &error) {
             FAIL() << "MutableStorage: " << error.error;
           });
-      ms->apply(block1, [](const auto &blk, auto &query, const auto &top_hash) {
+      auto bl = shared_model::proto::from_old(block1);
```

Comment:
I think old block should be renamed to `old_block...` and new one to `block` instead of 'bl'